libbeat/kibana/client.go

```diff
@@ -118,7 +118,7 @@ func NewClientWithConfig(config *ClientConfig) (*Client, error) {
 	}
 
 	log := logp.NewLogger("kibana")
-	log.Info("Kibana url: %s", kibanaURL)
+	log.Infof("Kibana url: %s\n", kibanaURL)
```

Comment:
Newline is not needed.
```suggestion
	log.Infof("Kibana url: %s", kibanaURL)
```